lib/vsntool.ex

```diff
@@ -13,15 +13,21 @@ defmodule Vsntool do
   }
   @shortcuts Map.keys(@options)
 
-  def main([]) do
-    IO.puts(version_from_git())
+  def main([]), do: execute("current_version")
+
+  def main([option]), do: execute(option)
```

Comment:
I would add a guard here as well to make sure that the command is correct. Otherwise `execute` gives a FunctionClauseError.

```
@commands Map.values(@options)
```
and the guard `when option in @options`

Reply:
Nevermind, did it myself :-) Thanks again!